torch/csrc/jit/passes/mkldnn_rewrite.h

```diff
@@ -3,6 +3,7 @@
 #include <ATen/Config.h>
 #include <torch/csrc/jit/api/module.h>
 #include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/ir/subgraph_matcher.h>
```

Comment:
Nit: you don't need to include this header in `mkldnn_rewrite.h`. Including it in `mkldnn_rewrite.cpp` should suffice.

Reply:
Yes, cleared.